nbproject/project.properties

```diff
@@ -0,0 +1,3 @@
+files.encoding=UTF-8
```

Comment:
I know it doesn't hurt, but should the nbproject folder really under version control?

Reply:
Since I don't see other IDE meta-data, I can push a change that ignores the whole folder.

If I may make my case, my mind was changed when I was introduced to `gitignore.io`'s templates for IntelliJ, NetBeans, and many others. Now, I could see the logic that went into the exclusions provided by the templates. For example, `.idea/codeStyle.xml` was not excluded. My interest piqued and on inspection the logic made sense: you want to share the code style with your peers, not have your peers be instructed to set a style or have a commit or pipeline check "correct them."

Finally, note that`.nbproject/private` is excluded. IMHO there are aspects of many IDE meta-data folders that should be version controlled, but are generally excluded as a whole because determining which aspects should be excluded (ignored) and other aspects that should be included is necessarily difficult to dedicate time too. gitignore.io really changed my outlook.

Reply:
@atomfrede removed the NB meta-data and updated the exclusions.